packages/trader/src/Modules/Contract/Components/InfoBox/info-box-longcode.tsx

```diff
@@ -22,20 +26,40 @@ const InfoBoxLongcode = ({ contract_info }: TInfoBoxLongcode) => {
                 <Text
                     size='xs'
                     className={classNames('info-box-longcode-text', {
-                        'info-box-longcode-text--collapsed': is_collapsed,
+                        'info-box-longcode-text--collapsed': is_collapsed || is_mobile,
                     })}
                 >
                     {contract_info.longcode}
```

Comment:
@akmal-deriv 
we can introduce a reusable variable:
`const is_collapsible_longcode = contract_info?.longcode && contract_info.longcode.length > max_longcode_length;`
and use it to fix iphone SE issue here :)
```suggestion
                    {is_collapsible_longcode && is_collapsed ? contract_info.longcode.slice(0, max_longcode_length) : contract_info.longcode}
```

Reply:
I think there's some confusion here, I did not introduce max_longcode_length as the exact amount of characters which will constantly represent 2,3, etc lines on the screen. This is not possible since each language has different characters of different width and all characters are not equal in the fonts we use. We shouldn't add any additional logic that would introduce new bugs and reduce usability and UX for users and screen readers at any point just to fix Iphone SE which is long abandoned anyways.

Reply:
@akmal-deriv last time I was working on this, i checked with our designer who confirmed that the message was supposed to be broken by the exact number of symbols and it didn't matter if it broke in the middle of the word since it's ending with '...' when collapsed. Hence, it doesn't matter which language it is. :)

Reply:
Just number of lines, breaking by symbols will cause issues in the future as I mentioned, so I don't wanna introduce new issues. It's not relevant anymore as the issue on iphone SE should be fixed with css.

Reply:
I put the number of characters just for convenience to place the button underneath, it was never meant to be used for breaking or similar things and is not any exact number but an approximate number to appear `somewhere` in the last line.